.ci.yaml

```diff
@@ -71,6 +71,22 @@ platform_properties:
       os: Linux
       device_type: "SM-A025V"
   mac:
+    properties:
+      dependencies: >-
+        []
+      os: Mac-12
+      device_type: none
+      cpu: arm64
```

Comment:
First, attempting to run these tests only on arm64 to confirm they all pass on that arch.
(This change will not be checked in, just confirming on this PR that all tasks pass on arm64 as well as x64).